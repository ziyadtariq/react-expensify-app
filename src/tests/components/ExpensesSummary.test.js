import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';

test('should render ExpensesSummary with single expense correctly', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with multiple expense correctly', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={10235}/>);
    expect(wrapper).toMatchSnapshot();
});