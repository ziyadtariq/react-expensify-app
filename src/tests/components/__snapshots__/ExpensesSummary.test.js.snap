// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with multiple expense correctly 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $102.35
  </h1>
</div>
`;

exports[`should render ExpensesSummary with single expense correctly 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;
